server/channels/app/export.go

```diff
@@ -163,6 +171,19 @@ func (a *App) BulkExport(ctx request.CTX, writer io.Writer, outPath string, job
 	return nil
 }
 
+func (a *App) exportAttachments(ctx request.CTX, attachments []imports.AttachmentImportData, outPath string, zipWr *zip.Writer, totalExportedFiles *int, totalFiles int) *model.AppError {
+	for _, attachment := range attachments {
+		if err := a.exportFile(outPath, *attachment.Path, zipWr); err != nil {
+			return err
```

Comment:
Not strictly related to this PR: Should the export continue if one file fails?

Reply:
I thought the same but that is out of scope for this PR